DevSkim-DotNet/Microsoft.DevSkim.CLI/Commands/SuppressionCommand.cs

```diff
@@ -0,0 +1,126 @@
+
+                    listOfReplacements
+                    .Sort((a, b) => a.Region.StartLine - b.Region.StartLine);
+
+                    var distinctReplacements  = listOfReplacements
+                    .GroupBy(x => x.Region.StartLine)
+                    .Select(x => x.FirstOrDefault());
+
+                    var ruleIds = string.Join(",", resultGroup.Select(x => x.RuleId).Distinct());
+
+                    if (File.Exists(potentialPath))
+                    {
+                        var theContent = File.ReadAllText(potentialPath).Split(Environment.NewLine);
+                        int currLine = 0;
+                        var sb = new StringBuilder();
+                        foreach (var replacement in distinctReplacements)
+                        {
+                            var zbStartLine = theContent[0] == string.Empty ? replacement.Region.StartLine: replacement.Region.StartLine - 1;
+                            var isMultiline = theContent[zbStartLine].EndsWith(@"\");
+                            var ignoreComment = $"{getPrefixComment(replacement.Region.SourceLanguage)} DevSkim: ignore {ruleIds} {getSuffixComment(replacement.Region.SourceLanguage)}";
+                            
+                            foreach (var line in theContent[currLine..zbStartLine])
+                                    sb.Append($"{line}{Environment.NewLine}");
+                            
+                            var suppression = isMultiline ? $"{ignoreComment}{theContent[zbStartLine]}{Environment.NewLine}" : 
+                            $"{theContent[zbStartLine]}{ignoreComment}{Environment.NewLine}";
+                            sb.Append(suppression);
+
+                            currLine = zbStartLine + 1;
+                        }
+
+                        foreach (var line in theContent[currLine..^1])
+                            sb.Append($"{line}{Environment.NewLine}");
+                        sb.Append($"{theContent.Last()}");
+
+                        if (!_opts.DryRun)
+                        {
+                            File.WriteAllText(potentialPath, sb.ToString());
+                        }
+                    }
+                    else
+                    {
+                        Console.Error.WriteLine($"{potentialPath} specified in sarif does not appear to exist on disk.");
+                    }
+                }
+            }
+
+            return (int)ExitCode.NoIssues;
+        }
+
+        private string getInlineComment(string language)
+        {
+           Languages devSkimLanguages = DevSkimLanguages.LoadEmbedded();
+           return devSkimLanguages.GetCommentInline(language);
+        }
+
+        private string getSuffixComment(string language)
+        {
+           Languages devSkimLanguages = DevSkimLanguages.LoadEmbedded();
+           return devSkimLanguages.GetCommentSuffix(language);
+        }
+
+        private string getPrefixComment(string language)
```

Comment:
Same as above.

Reply:
This method seems like excessive abstraction since its just a direct call to the devskim languages method. Suggest refactoring to just call the devskimlanguages api instead and remove this.